SoftLayer/CLI/file/count.py

```diff
@@ -0,0 +1,55 @@
+"""List number of file storage volumes per datacenter."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import columns as column_helper
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+COLUMNS = [
+    column_helper.Column('Datacenter',
+                         ('serviceResource', 'datacenter', 'name'),
+                         mask="serviceResource.datacenter.name"),
+    column_helper.Column('Count',
+                         '',
+                         mask=None)
+]
+
+DEFAULT_COLUMNS = [
+    'Datacenter',
+    'Count'
+]
+
+
+@click.command()
+@click.option('--datacenter', '-d', help='Datacenter shortname')
+@click.option('--sortby', help='Column to sort by', default='Datacenter')
+@click.option('--columns',
+              callback=column_helper.get_formatter(COLUMNS),
+              help='Columns to display. Options: {0}'.format(
+                  ', '.join(column.name for column in COLUMNS)),
+              default=','.join(DEFAULT_COLUMNS))
+@environment.pass_env
+def cli(env, sortby, columns, datacenter):
+    """List number of file storage volumes per datacenter."""
+
+    file_manager = SoftLayer.FileStorageManager(env.client)
+    file_volumes = file_manager.list_file_volumes(datacenter=datacenter,
+                                                  mask=columns.mask())
```

Comment:
instead of modifying the mask in the manager, just do something like this. fix the line width so tox doesn't complain about it too.

```
    mask = "mask[serviceResource[datacenter[name]],replicationPartners[serviceResource[datacenter[name]]]]"
    block_volumes = block_manager.list_block_volumes(datacenter=datacenter,
                                                     mask=mask)
```